npm v3 still doesn't work

Are you sure [this PR](https://github.com/Financial-Times/origami-build-tools/pull/330) actually fixed it? For me I still get this error with the latest master of obt:

```
{ [Error: ModuleNotFoundError: Module not found: Error: Cannot resolve 'file' or 'directory' /Users/callum.locke/code/ft/project-starter-kit/node_modules/babel-runtime/helpers/interop-require-default in /Users/callum.locke/code/ft/project-starter-kit/client/scripts]
  message: 'ModuleNotFoundError: Module not found: Error: Cannot resolve \'file\' or \'directory\' /Users/callum.locke/code/ft/project-starter-kit/node_modules/babel-runtime/helpers/interop-require-default in /Users/callum.locke/code/ft/project-starter-kit/client/scripts',
  showStack: false,
  showProperties: true,
  plugin: 'webpack-stream',
  __safety: { toString: [Function: bound ] } }
```

Using npm 3. @matthew-andrews is it possible you tested this with an npm link'd copy of obt (which would mean a particular directory structure so it wouldn't necessarily work the same)?

--- a/lib/plugins/babelRuntimePathResolver.js
+++ b/lib/plugins/babelRuntimePathResolver.js
@@ -1,13 +1,10 @@
-// Makes paths to babel-runtime polyfills absolute as they're
-// in OBT's node_modules directory and not the module's
+// Makes paths to babel-runtime polyfills absolute as they might be in OBT's
+// node_modules directory and not the module's, depending on node version
 'use strict';
 
 const falafel = require('falafel');
 
 const prefix = /babel-runtime/;
-
-// HACK: Can't require.resolve babel-runtime so use babel-core instead
-const rootDirectory = require.resolve('babel-core').replace(/(.*node_modules).*/, '$1');
 
 module.exports = function babelRuntimePathResolver(file) {
 	this.cacheable = true;
@@ -16,7 +13,7 @@ module.exports = function babelRuntimePathResolver(file) {
 		// Find require() calls
 		if (node.type === 'CallExpression' && node.callee.type === 'Identifier') {
 			if (node.callee.name === 'require' && prefix.test(node.arguments[0].value)) { // require("babel-runtime/*")
-				const requirePath = rootDirectory + '/' + node.arguments[0].value;
+				const requirePath = require.resolve(node.arguments[0].value);
 				node.update('require(' + JSON.stringify(requirePath) + ')');
 			} else {// none of the above, skip
 				return;

